lib/chef/util/dsc/local_configuration_manager.rb

```diff
@@ -78,6 +78,18 @@ def lcm_command_code(configuration_path, test_only_parameters)
 EOH
     end
 
+    def test_lcm_command_code
+      ps_version = Mixlib::ShellOut.new("powershell $PSVersionTable.PSVersion.Major").run_command
```

Comment:
*or* use the query helper that is already there: ```Chef::Platform.supported_powershell_version?(node, 5)```

Reply:
Thanks @stuartpreston,